content/en/docs/about/community.md

```diff
@@ -137,10 +137,6 @@ In addition, please check out the community-curated [awesome list of projects an
 
 The Kubeflow trademark and logos are registered trademarks of The Linux Foundation® (TLF), please review the [Kubeflow Brand Guidelines](https://www.linuxfoundation.org/legal/trademark-usage) for more information.
 
-## Kubeflow Steering Committee
-
-The [Kubeflow Steering Committee (KSC)](https://github.com/kubeflow/community/blob/master/KUBEFLOW-STEERING-COMMITTEE.md) is the governing body of the Kubeflow project, providing decision-making and oversight pertaining to the Kubeflow project policies, sub-organizations, and financial planning, and defines the project values and structure.
-
 ## Kubeflow Working Groups
```

Comment:
@franciscojavierarceo Please can you migrate this part to the governance page as well ?

Reply:
It's # 4.

Reply:
In that case, can you remove this `Kubeflow Working Groups` section from the `community.md` page ?

Reply:
I didn't because it lists the WG Meetings and I felt that made sense to keep in the community page.